cookbooks/redis/attributes/overrides.rb

```diff
@@ -0,0 +1,3 @@
+# Overrides default Redis version with the latest available 
```

Comment:
Why are you creating this file? It's sufficient to set the version in `cookbooks/redis/attributes/default.rb`

Reply:
Just thought to arrange them up with abstracting some most changeable features. But let me remove that file and move with only the `default.rb`